docs/source/index.rst

```diff
@@ -638,7 +638,11 @@ When using WinPixEventRuntime version 1.0.210209001:
 
   #if defined(USE_PIX) || !defined(PIX_XBOX)
   #define PIX_CONTEXT_EMIT_CPU_EVENTS
+  #endif
+  
+  #ifndef PIX_XBOX
```

Comment:
Seems like we should still check USE_PIX here to avoid including the header file when USE_PIX is not defined?

Could either change line 643 to:

  `#if defined(USE_PIX) && !defined(PIX_XBOX)`

or change the entire block to:

 ```
#if defined(USE_PIX) || !defined(PIX_XBOX)
  #define PIX_CONTEXT_EMIT_CPU_EVENTS
  
  #ifndef PIX_XBOX
  #include "AmdDxExt\AmdPix3.h"
  #define PIX_AMD_EXT
  #endif
#endif
```

Reply:
You're totally right, I went ahead with your 2nd suggestion